pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -3724,10 +3781,11 @@ func TestInterPodAffinityWithMultipleNodes(t *testing.T) {
 			},
 			nodesExpectAffinityFailureReasons: [][]algorithm.PredicateFailureReason{
 				{ErrPodAffinityNotMatch, ErrPodAntiAffinityRulesNotMatch},
+				{ErrPodAffinityNotMatch, ErrPodAntiAffinityRulesNotMatch},
 			},
 			fits: map[string]bool{
 				"nodeA": false,
-				"nodeB": true,
+				"nodeB": false,
```

Comment:
the change is the same as https://github.com/kubernetes/kubernetes/pull/68173/commits/d20e5e3e699cf927ba29552f7b735159cbe868d5#r217482459

Reply:
and I can delete this case as it's duplicate as https://github.com/kubernetes/kubernetes/pull/68173/commits/d20e5e3e699cf927ba29552f7b735159cbe868d5#r217482459